src/paste3/model_selection.py

```diff
@@ -144,95 +144,6 @@ def calculate_convex_hull_edge_inconsistency(sliceA, sliceB, pi):
     return measure_A, measure_B
 
 
-"""
-Main function.
-"""
-
-
-def select_overlap_fraction(sliceA, sliceB, alpha=0.1):
```

Comment:
`select_overlap_fraction_plotting` was simply a copy of this function with one additional parameter (`show_plot`) which only adds to the functionality as an added side-effect, and doesn't modify the rest of the function's behavior. So that has been renamed to `select_overlap_fraction`, and this function is gone. The associated test for the function is gone too.